src/cheevos/Cheevos.h

```diff
@@ -51,6 +58,9 @@ class CCheevos
   size_t Reduce(size_t addr, size_t mask);
 
   static unsigned int PeekInternal(unsigned address, unsigned num_bytes, void* ud);
+  static void RuntimeEventHandler(const rc_runtime_event_t* runtime_event);
+
+  void (*m_callback)(const char* achievement_url, unsigned cheevo_id);
```

Comment:
Not sure why but I remember that we wanted to remove this, I had open this PR https://github.com/Shardul555/game.libretro/pull/1 I just rebased it on this PR. 

@garbear can you check it if you remember why we wanted this

Reply:
We don't want a static function in RetroPlayer.